plugins/module_utils/openssh/utils.py

```diff
@@ -18,7 +18,6 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 import re
```

Comment:
You need to keep an empty shell of this file in the old location which imports `parse_openssh_version` from the new version to avoid breaking backwards compatibility (module_utils is also part of the public API of the collection).

Reply:
Added the original back with just the import in the latest commit.